.travis.yml

```diff
@@ -0,0 +1,10 @@
+os:
+  - linux
+
+services:
+  - docker
+
+before_install:
+  - docker build -t gary29198/burrow:latest .
+  - docker login -u gary29198 -p 0001221149
```

Comment:
PR aside, you should probably rotate your password ASAP.

Reply:
really sorry... i was just teaching one of my friend to raise a PR and accidentally ended up raising to your repo.... really very sorry

Reply:
Haha, no worries, happens to the best of us! 🙌